### ISSUE TITLE
stick can not be installed.

### ISSUE THREAD

**Issue by nilshero** [ringo/stick#8]
Hi,
tried to install stick on ringo under Win XP:
### a) based on the binary RingoJS v0.7.0 tar.gz archive

downloaded ringojs-0.7.tar.gz from [the ringo website](http://ringojs.org/downloads).
I get this log, this is no problem because there is no ringo/jsgi directory:

``` batch
Microsoft Windows XP [Version 5.1.2600]
(C) Copyright 1985-2001 Microsoft Corp.

C:\ringo\bin>ringo-admin install hns/stick
Downloading http://github.com/hns/stick/zipball/master
Installing package stick
 + C:\ringo\bin\\packages\stick\LICENSE
(... downloads everything ...)
 + C:\ringo\bin\\packages\stick\update-docs.sh
Done
Error: failed to remove file C:\systemp\nils\ringo-install1572099501524327036.zip (fs.js#460)

C:\ringo\bin>cd packages\stick

C:\ringo\bin\packages\stick>ringo demo.js
Wrapped java.io.IOException: Resource "ringo/jsgi/response" not found or not readable (stick/middleware/static.js#12)
        at stick/middleware/static.js:12
        at stick/middleware.js:46
        at stick.js:19
        at C:\ringo\bin\packages\stick\demo.js:4


C:\ringo\bin\packages\stick>
```
### b) based on the RingoJS master tar.gz git snapshot

downloaded ringo-ringojs-v0.7.0-85-g11170e4.tar.gz from [the same website](http://ringojs.org/downloads).
I get this log, i do not understand this, because C:/ringo/bin/packages/stick/lib/stick exists:

``` batch
Microsoft Windows XP [Version 5.1.2600]
(C) Copyright 1985-2001 Microsoft Corp.

C:\ringo\bin>ant jar
Buildfile: C:\ringo\bin\build.xml

init:

compile:
    [mkdir] Created dir: C:\ringo\bin\build\classes
    [javac] Compiling 48 source files to C:\ringo\bin\build\classes

jar:
      [jar] Building jar: C:\ringo\bin\lib\ringo.jar
      [jar] Building jar: C:\ringo\bin\run.jar

BUILD SUCCESSFUL
Total time: 2 seconds

C:\ringo\bin>ringo-admin install hns/stick
Downloading http://github.com/hns/stick/zipball/master
Installing package stick
 + C:\ringo\bin\\packages\stick\LICENSE
(... downloads everything ...)
 + C:\ringo\bin\\packages\stick\update-docs.sh
Done
Error: failed to remove file C:\systemp\nils\ringo-install6510812854526382940.zip (fs.js#480)

C:\ringo\bin>cd packages\stick

C:\ringo\bin\packages\stick>ringo demo.js
Cannot find module 'stick' (C:\ringo\bin\packages\stick\demo.js#4)
        at C:\ringo\bin\packages\stick\demo.js:4


C:\ringo\bin\packages\stick>
```
### c) i tried it with a fresh clone direct from gitHub

behaves like b)

In short, i was not able to install stick.

Greets, Nils-Hero
